segmentation/utils/data_utils.py

```diff
@@ -109,6 +115,9 @@ def load_imgs_from_dir(data_dir, img_sub_folder=None, fovs=None, imgs=None,
     if len(imgs) == 0:
         raise ValueError("No imgs found in designated folder")
 
+    # remove .tif or .tiff from image name
+    img_names = [os.path.splitext(img)[0] for img in imgs]
+
```

Comment:
Is there a reason these lines were moved from 145-147 to here?

Reply:
pickybacking on the already-written code of the function was made easier by essentially making `fovs` act like `imgs`, instead of having `fovs = [""]` and using imgs the same.  So theres a point where I move `imgs` into `fovs` and redefine `imgs`, so `img_names` needed to be set earlier.  In hindsight, I think I could've changed it to:
```
img_names = [os.path.splitext(fov)[0] for fov in fovs]
```
and left it where it was, but there's not much of a difference.  No matter now tho, since I'll probably move this jazz into a separate function.

tl;dr makes piggy backing on already written code easier if it's declared earlier

Reply:
Ah OK, I see.